extensions/machine-learning-services/src/common/constants.ts

```diff
@@ -164,8 +167,15 @@ export const updateModelFailedError = localize('models.updateModelFailedError',
 export const importModelFailedError = localize('models.importModelFailedError', "Failed to register the model");
 export const loadModelParameterFailedError = localize('models.loadModelParameterFailedError', "Failed to load model parameters'");
 export const unsupportedModelParameterType = localize('models.unsupportedModelParameterType', "unsupported");
-
-
+export const dashboardTitle = localize('dashboardTitle', "SQL ML");
```

Comment:
nit : SQL Machine Learning sounds clearer than just ML.. but if we have closed thats ok

Reply:
@MsSQLGirl is working on finalizing the name